sections/cart-drawer.liquid

```diff
@@ -0,0 +1,350 @@
+<script src="{{ 'cart.js' | asset_url }}" defer="defer"></script>
+
+<style>
+  .drawer {
+    visibility: hidden;
+  }
+</style>
+
+<cart-drawer class="drawer{% if cart == empty %} is-empty{% endif %}">
+  <div class="cart-drawer" id="cart-drawer">
```

Comment:
This `id` is also not using the same pattern we've been using on other parts of the theme.

Should we rename it to `CartDrawer`?

```suggestion
  <div id="CartDrawer" class="cart-drawer">
```